scripts/benchmarking/run_all_benchmarks.sh

```diff
@@ -10,6 +10,33 @@
 #
 # Should be run from the root of the repo.
 
+# Steps and repeats for main benchmark.
+BENCHMARK_STEPS=50
+BENCHMARK_REPEAT=20
+
+# Steps and repeats for benchmarking so called "one-time extrinsics", 
+# which may be called only once and require a different benchmarking approach with more repeats.
+BENCHMARK_STEPS_ONE_TIME_EXTRINSICS=2
+BENCHMARK_REPEAT_ONE_TIME_EXTRINSICS=1000
+
+# List of one-time extrinsics to benchmark. This is only used for pallet_gear.
+ONE_TIME_EXTRINSICS=(
+  "gr_reply"
+  "gr_reply_wgas"
+  "gr_reply_commit"
+  "gr_reply_commit_wgas"
+  "gr_reply_input"
+  "gr_reply_input_wgas"
+  "gr_reservation_reply"
+  "gr_reservation_reply_commit"
```

Comment:
`gr_reservation_reply_commit` is duplicated here

maybe we can get `ONE_TIME_EXTRINSICS` using the `grep`?
```
cat pallets/gear/src/benchmarking/mod.rs | grep "0 .. 1;" -B 1 | grep -E "{$" | awk '{print $1}'
```

Reply:
Super cool, thx!